### PR TITLE
SOL-1266: Style default sparks

### PR DESCRIPTION
**Pull request by dmarkova**

### CODE REVIEW

File: event.html [hunk 1]
```diff
@@ -105,72 +113,72 @@ <h3 data-reactid=".832vlajev4.1.1.1.$2.$0.1">Påskemoro med barna</h3>
...
+</script><script src="/javascript/app.js"></script>
 </body>
 </html>
```
Comment (misund) [inline event.html:184]: Do these changes include anything else than formatting? If not, why are they not in a separate commit?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sol-style-guide",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (misund) [inline package.json:3]: Please do not publish unmerged changes.

File: scss/settings/_colors.scss [hunk 3]
```diff
@@ -32,3 +32,8 @@ $menuButtonActiveColor: lighten($headerColor,10%);
 $menuFirstLevelBgColor: $offwhiteColor;
 $menuSecondLevelBgColor: #e6e6e6;
 $extraMenuBorder: $offwhiteColor;
+
+/*Events Colors*/
```
Comment (misund) [inline scss/settings/_colors.scss:36]: Why do events need their own colors? The purpose of this repo is to standardize and ease maintainability.

File: event.html (hunk 1 above)
Comment (dmarkova) [inline event.html:184]: I don't know why my editor changed formatting. 
But I've changed the html markup of the default spark here.

File: scss/modules/_typography-fonts.scss [hunk 4]
```diff
@@ -25,3 +25,6 @@ $headingFont: 'Roboto Slab', 'serif';
  * }
  */
 $mainFont: 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:800");
+$headingEventFont: 'Open Sans', 'sans-serif';
```
Comment (misund) [inline scss/modules/_typography-fonts.scss:30]: As it is written, it seems that these two lines are meant only for events. But the typography module is loaded in all projects.

If the above is correct, these changes will introduce bloat in every other project but events.

File: scss/settings/_colors.scss (hunk 3 above)
Comment (dmarkova) [inline scss/settings/_colors.scss:36]: Because the design is different, no?

File: scss/modules/_typography-fonts.scss (hunk 4 above)
Comment (dmarkova) [inline scss/modules/_typography-fonts.scss:30]: Should it be moved to event.cscc?

File: scss/event.scss [hunk 5]
```diff
@@ -1,47 +1,133 @@
...
+  }
+
+}
+
+
+[class^="item-"] {
+  display:block;
+  position:relative;
+  margin:0;
+  padding: 2.4rem 1.48331rem;
```
Comment (misund) [inline scss/event.scss:44]: Padding values should use parameters.

File: scss/event.scss [hunk 6]
```diff
@@ -1,47 +1,133 @@
...
+  border-radius: 0;
+
+  &,
+  &:hover,
+  &:active,
+  &:focus {
+    background: $eventBgColor;
```
Comment (misund) [inline scss/event.scss:51]: Colors should probably not be event specific.

File: scss/event.scss [hunk 7]
```diff
@@ -1,47 +1,133 @@
...
+    margin: 0 0 5px;
+    padding:0;
+    font-weight: 800;
+    font-size: 2.1rem;
+    color: $headingColor;
+    font-family: $headingEventFont;
+    line-height: 2.5rem;
+  }
+  p {
+    margin-bottom:1rem;
```
Comment (misund) [inline scss/event.scss:75]: Breaks vertical rhythm. Vertical margins should be a multiple of $verticalBase.

File: scss/event.scss [hunk 8]
```diff
@@ -1,47 +1,133 @@
...
+    height: auto;
+  }
+  h3 {
+    margin: 0 0 5px;
+    padding:0;
+    font-weight: 800;
+    font-size: 2.1rem;
+    color: $headingColor;
+    font-family: $headingEventFont;
+    line-height: 2.5rem;
```
Comment (misund) [inline scss/event.scss:72]: Breaks vertical rhythm. Line-heights should be a multiple of $verticalBase.

File: scss/event.scss [hunk 9]
```diff
@@ -1,47 +1,133 @@
...
+    max-width: 100%;
+    height: auto;
+  }
+  h3 {
+    margin: 0 0 5px;
+    padding:0;
+    font-weight: 800;
+    font-size: 2.1rem;
+    color: $headingColor;
+    font-family: $headingEventFont;
```
Comment (misund) [inline scss/event.scss:71]: Why is this event-specific?

File: scss/event.scss [hunk 10]
```diff
@@ -1,47 +1,133 @@
...
+    border-top: 2px solid $eventBorderColor;
+  }
+
+  img {
+    display: block;
+    max-width: 100%;
+    height: auto;
+  }
+  h3 {
+    margin: 0 0 5px;
```
Comment (misund) [inline scss/event.scss:66]: Breaks vertical rhythm. The sum of vertical margins should be a multiple of $verticalBase.

File: scss/event.scss [hunk 11]
```diff
@@ -1,47 +1,133 @@
...
+  }
+}
+
+.item-default {
+  display: table;
+  width: 100%;
+  table-layout: fixed;
+
+  .img-wrap,
+  .text-wrap {
```
Comment (misund) [inline scss/event.scss:85]: Wrapper stylings require bloated markup. These should be rewritten.

File: scss/event.scss (hunk 10 above)
Comment (dmarkova) [inline scss/event.scss:66]: Seriously? Can you make documentation, how to write your css? :)

File: event.html (hunk 1 above)
Comment (misund) [inline event.html:184]: Ah, I see. A bit hard to figure out the actual changes from the indentation ones... when all the lines are changed..

File: scss/settings/_colors.scss (hunk 3 above)
Comment (misund) [inline scss/settings/_colors.scss:36]: The design should be consistent.

File: scss/modules/_typography-fonts.scss (hunk 4 above)
Comment (misund) [inline scss/modules/_typography-fonts.scss:30]: I'm not sure. Is there any reason why the default typography should be different in events than in other projects?

File: scss/event.scss (hunk 10 above)
Comment (misund) [inline scss/event.scss:66]: Yes, we should probably invest the time to describe the basic principles in this repo. I'll see if I can find sometime today already.

File: scss/modules/_typography-fonts.scss (hunk 4 above)
Comment (dmarkova) [inline scss/modules/_typography-fonts.scss:30]: As I see, other projects in style guide don't use Open Sans. 
So should I make Open Sans default font for headings? Or should I remove it and use Roboto, as it was before?

**Pull request closed without merging**